android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -748,9 +739,6 @@ private void initializePlayerCore(ReactExoplayerView self) {
 
         PlaybackParameters params = new PlaybackParameters(rate, 1f);
         player.setPlaybackParameters(params);
-        if (analyticsMeta != null) {
```

Comment:
we dont need this anymore, we need to delay the youbora init after `player.prepare` hence removed this